appveyor.yml

```diff
@@ -52,10 +52,24 @@ after_build:
   - ps: |
       cd c:\projects\Revive\Installer
       $env:Path += ";C:\Program Files (x86)\NSIS"
-      makensis /DDEV_BUILD Revive.nsi
+      If ($env:APPVEYOR_REPO_TAG -eq "false" -Or $env:APPVEYOR_REPO_TAG_NAME -like "*beta") {
+        makensis /DDEV_BUILD Revive.nsi
+      } Else {
+        makensis Revive.nsi
+      }
 
 artifacts:
   - path: Installer\ReviveInstaller.exe
     name: Revive Installer
   - path: Release\Revive.zip
     name: Revive Build
+
+deploy:
+  provider: GitHub
+  auth_token:
+    secure: 1ckkLEAo2w5lZ4oziHqrXRQvuRwQoLuu9Q0ypuvk9CFp005B6T02JvV6aQdAGrHU

```

Comment:
This needs to be changed to a valid token for the LibreVR/Revive repository (see PR description).

Reply:
FYI, this token is visible to the public. You might want to use [github's built in secrets feature for the repo](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) to hide the token.

Reply:
Thanks for your concern. However, this token is already encrypted, see PR description. Unless you have access to the encrypting AppVeyor account, you can't do anything with it.